Mathlib/RingTheory/Trace.lean

```diff
@@ -190,8 +193,8 @@ variable (R S)
 /-- The `traceForm` maps `x y : S` to the trace of `x * y`.
 It is a symmetric bilinear form and is nondegenerate if the extension is separable. -/
 noncomputable def traceForm : BilinForm R S :=
-  -- Porting note: dot notation `().toBilin` does not work anymore.
-  LinearMap.toBilin (LinearMap.compr₂ (lmul R S).toLinearMap (trace R S))
+-- Porting note: dot notation `().toBilin` does not work anymore.
```

Comment:
```suggestion
```
This does not make sense anymore.

Reply:
`traceForm` got redefined in #12078, but the porting note remains.

Reply:
Well, I guess it's time to remove it.

Reply:
#12489